plugins/doc-explorer-classic/src/index.ts

```diff
@@ -0,0 +1,20 @@
+export * from './FieldDoc';
+export { DocExplorer } from './DocExplorer';
+
+const ID = 'doc-explorer-classic';
+
+import DocExplorerGraphiQL from './DocExplorerGraphiQL';
```

Comment:
Is this a wrong path. Because I can't find it

Reply:
good catch!